pom.xml

```diff
@@ -495,6 +495,33 @@
                     <lineEnding>LF</lineEnding>
                 </configuration>
             </plugin>
+            <plugin>
+                <groupId>io.sundr</groupId>
+                <artifactId>sundr-maven-plugin</artifactId>
+                <version>0.200.0</version>
+                <inherited>false</inherited>
+                <configuration>
+                    <boms>
+                        <bom>
+                            <artifactId>smallrye-graphql-bom</artifactId>
+                            <name>SmallRye: GraphQL Bill Of Materials</name>
+
+                            <modules>
+                                <includes>
+                                    <include>io.smallrye:*</include>
```

Comment:
We should exclude test-related modules (`smallrye-graphql-integration-tests`, `smallrye-graphql-client-generator-test`, `smallrye-graphql-maven-plugin-tests`, `smallrye-graphql-tck`, `smallrye-graphql-client-tck`) and also `smallrye-graphql-documentation`, so maybe it would be better to explicitly list the modules to include (without a wildcard)

Reply:
DONE.

Went with exclusions instead of inclusions cause:

a) you dont want to forget a new module when a release comes out, and people do forget to update things that just work
b) is less harmful to have an extra dependency you don't need managed then it is to not have it at all

Honestly, I would keep the test dependencies.  Anyone using the test dependency will need to manually declare a version for that one.


But, that is your call.

Reply:
We don't want to publish these modules at all (I don't see a reason why anybody would depend on them), so they shouldn't be in the bom either.